modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -752,6 +756,48 @@ public void testIdleVerifyOnInactiveClusterWithPersistence() throws Exception {
         assertContains(log, testOut.toString(), "The check procedure has finished, no conflicts have been found.");
     }
 
+    /**
+     *
+     */
+    @Test
+    public void testIdleVerifyCancelCommand() throws Exception {
+        final int gridsCnt = 4;
+
+        IgniteEx srv = startGrids(gridsCnt);
+
+        srv.cluster().state(ACTIVE);
+
+        IgniteCache<Integer, Integer> cache = srv.createCache(DEFAULT_CACHE_NAME);
```

Comment:
Default backup count are 0.
Which means idle_verify will run on one node only.
Let's configure `gridsCnt == CacheConfiguration.backup == 3` in this test.